class DropboxAuthError(Exception):
    pass


class DropboxWebsiteError(Exception):
    pass
